RecoParticleFlow/PFProducer/src/PFEGammaAlgo.cc

```diff
@@ -2023,10 +2023,46 @@ fillPFCandidates(const pfEGHelpers::HeavyObjectCache* hoc,
     // build the refined supercluster from those clusters left in the cand
     refinedscs_.push_back(buildRefinedSuperCluster(RO));
 
-    // forward the time from the seed cluster
-    if (!RO.ecalclusters.empty()) {
-        auto const & seedPFClust = *RO.ecalclusters.front().first->clusterRef();
-        cand.setTime( seedPFClust.time(), seedPFClust.timeError() );
+    //*TODO* cluster time is not reliable at the moment, so only use track timing
+    if (false) {
```

Comment:
make a config parameter instead?
If not, since this needs recompilation anyways, perhaps put it in compile macro so that one or the other can be chosen in recompile for tests without editing the code.

Reply:
This can't be re-enabled without other extensive code changes at the calo local reco level anyways, so I don't have too strong feelings about it, just wanted to make sure we don't forget all the places where it will need to be added back.

Reply:
Will the interface or implementation change when it is to be enabled?
If you just need to mark the places where something should be added, just add a comment but no code, which still needs to be compiled [I doubt that the compiler completely ignores what's inside ```if (false) ```]
An ```#ifdef``` is perhaps a cleaner way to mark the territory.

Reply:
I don't think the interface will change at this level.  The issues are with how the times are actually assigned to the clusters and/or underlying rechits.

Reply:
I don't think the interface will change at this level.  The issues are with how the times are actually assigned to the clusters and/or underlying rechits.

Reply:
a config parameter in this case may be more practical.
If there is still an issue upstream, it's not obvious it will be fixed once and forever. So, the future may need it on or off.